WorkloadManagementSystem/Client/JobState/JobState.py

```diff
@@ -18,12 +18,32 @@
 
 class JobState(object):
 
+  class DBHold:
```

Comment:
I guess using an old style class was accidental?
```suggestion
  class DBHold(object):
```

Reply:
Updated